mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java

```diff
@@ -618,6 +618,10 @@ public void onStopTrackingTouch(SeekBar seekBar) {
                             spinnerSelectedIndex = spinnerArray.size() - 1;
                         }
                 }
+                if (spinnerSelectedIndex == -1) {
+                    spinnerArray.add("");
```

Comment:
Maybe we should add "Please select...", "Undefined", "Value not defined in selection" or just "       ". The empty spinner is pretty hard to hit.

Reply:
Agreed!

Reply:
I would like to avoid adding a new string to translate.
Is it ok if I use 3 spaces ?

Reply:
What about 10 spaces? 3 sounds a little bit too less.

Reply:
> I would like to avoid adding a new string to translate.

If technicalities like this are hindering development we are doomed :) Did you consider using https://crowdin.com for the translation management?
Just spaces sounds unclear. Could you post a screenshot?

Reply:
See #342

Reply:
Translation management is certainly not the purpose of this PR.
My idea with this PR was to propose a kind of workaround which is better than nothing. I hope it could be better handled later (with the blank line being hidden) by a developer having the correct skills. That is why I wanted to keep it as simple as possible.
Additionally Basic UI is displaying a blank in this case. I think it is a good idea to have a similar behaviour in the Android app.
If you really want a value, we will probably discuss during ages about this value. I could propose "Other".

Reply:
When basic ui uses whitespace, then we also should use it. You might want to use non breaking spaces.

Reply:
"Value not defined in selection" is the right description but is largely too long.

Reply:
> I think it is a good idea to have a similar behaviour in the Android app.

Agreed. Okay then 👍